KsApi/mutations/inputs/CreateBackingInput.swift

```diff
@@ -4,31 +4,32 @@ public struct CreateBackingInput: GraphMutationInput {
   let amount: String
   let locationId: String?
   let paymentSourceId: String
-  let paymentType: String
```

Comment:
Removed `paymentType` because after chatted with the backend team, it's not *really* an input field (should always be set to `credit_card`). They plan on removing the field entirely in the near future.